package.json

```diff
@@ -49,7 +49,7 @@
     "extend": "=3.0.1",
     "ip": "=1.1.5",
     "json-schema": "=0.2.3",
-    "json-sql": "LiskHQ/json-sql#27e1ed1",
+    "json-sql": "SargeKhan/json-sql#decode-operator",
```

Comment:
Wasn't the changes merged already to LiskHQ/json-sql?

Reply:
Yes, I will update it to the LiskHQ version.

Reply:
How is it going? ;)